infra/terraform/test-org/org/locals.tf

```diff
@@ -743,6 +743,15 @@ locals {
       groups          = ["dee-data-ai", local.jss_common_group]
       enable_periodic = true
     },
+    {
+      name            = "terraform-genai-knowledge-base"
```

Comment:
@NimJay could you grant `GoogleCloudPlatform/blueprint-solutions` admin access

Reply:
Admin access has been granted to [the `GoogleCloudPlatform/blueprint-solutions` team](https://github.com/orgs/GoogleCloudPlatform/teams/blueprint-solutions).
(Thanks, @davidcavazos! :))

Reply:
Thanks!